pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -1,53 +1,59 @@
 { lib
 , buildPythonPackage
 , fetchPypi
-, fetchpatch
 , ipykernel
 , ipywidgets
-, pythonOlder
-, pytestCheckHook
-, pandas
 , jinja2
 , numpy
+, pandas
+, pytestCheckHook
+, pythonOlder
+, setuptools
 , traitlets
 }:
 
 buildPythonPackage rec {
   pname = "pydeck";
-  version = "0.7.1";
+  version = "0.8.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
-  patches = [
-    # fixes build with latest setuptools
-    (fetchpatch {
-      url = "https://github.com/visgl/deck.gl/commit/9e68f73b28aa3bf0f2a887a4d8ccd2dc35677039.patch";
-      hash = "sha256-YVVoVbVdY5nV+17OwYIs9AwKGyzgKZHi655f4BLcdMU=";
-      stripLen = 2;
-    })
-  ];
-
   src = fetchPypi {
     inherit pname version;
-    sha256 = "907601c99f7510e16d27d7cb62bfa145216d166a2b5c9c50cfe2b65b032ebd2e";
+    hash = "sha256-B+3egz98/O9nSRJDURlap9zSRmPUkJ/XiY29C2+8Aew=";
   };
 
-  pythonImportsCheck = [ "pydeck" ];
-
-  nativeCheckInputs = [ pytestCheckHook pandas ];
-
-  # tries to start a jupyter server
-  disabledTests = [ "test_nbconvert" ];
+  nativeBuildInputs = [
+    setuptools
+  ];
```

Comment:
Since you're here, would you mind also adding `wheel`? It's implicitly added by Python setup hooks at the moment, but I'm hoping that will change with some work I'm doing.

Reply:
I've added it.